src/geocoder/TECH_DEBT__TemporarilyCopied__Geocoder.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace GovUk.Education.SearchAndCompare.Geocoder
+{
+
+    /// <summary>
+    ///    Tech debt: this is copy-and-pasted from Geocoder.cs in search-and-compare-ui.
+    ///    We should extract a utilities project these two can share.
+    ///    https://trello.com/c/66mYU1OF/308-tech-debt-extract-geocoder-into-shared-utilities-nuget-package
+    /// </summary> 
+    public class TECH_DEBT__TemporarilyCopied__Geocoder
```

Comment:
Can we call this something sensible please and keep the tech debt thing in trello. The name is making my eyes bleed.

Reply:
Haha that's on purpose though. It should be a thorn in our sides while it lives